aips/aip-55.md

```diff
@@ -0,0 +1,97 @@
+---
+aip: 55
+title: Generalize Transaction Authentication and Support Arbitrary K-of-N MultiKey Accounts
+author: davidiw, hariri
```

Comment:
I don't think I can overwrite your commit, but will provide this as a suggested change
```suggestion
author: davidiw, hariria
```